crates/sequencer/src/sequencer.rs

```diff
@@ -0,0 +1,17 @@
+use crate::config::SequencerConfig;
+use blockifier::state::state_api::{State, StateReader};
+
+/// Sequencer is the main struct of the sequencer crate.
+/// Using a trait bound for the state allows for better
+/// speed, as the type of the state is known at compile time.
+pub struct Sequencer<S: State + StateReader> {
+    config: SequencerConfig,
```

Comment:
do we need to store the config?

Reply:
reminds me this https://www.thecodedmessage.com/posts/default-params/#scaling-defaults-in-rust-struct-update-syntax